hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServerImpl.java

```diff
@@ -338,6 +340,49 @@ public boolean addSCM(AddSCMRequest request) throws IOException {
     }
   }
 
+  public void updateRaftPeerPriority(String peerId) throws IOException {
+    List<RaftPeer> raftPeers =
+        new ArrayList<>(getDivision().getGroup().getPeers());
+    List<RaftPeer> oldPeers = new ArrayList<>(raftPeers);
+    RaftPeer peerToUpdate = raftPeers.stream()
+        .filter(peer -> peer.getId().toString().equals(peerId)).findFirst()
+        .get();
```

Comment:
`findFirst` returns an `Optional` object, which can be `null`. Calling `get` can result in `NoSuchElementException`.